src/components/ModalWindow/ModalWindow.js

```diff
@@ -0,0 +1,23 @@
+import './ModalWindow.scss';
+
+export default function ModalWindow() {
+  this.modal = document.createElement('div');
+  this.content = document.createElement('div');
+}
+
+ModalWindow.prototype.render = function (parent, content) {
+  this.modal.classList.add('modal');
+  this.content.classList.add('modal__content');
+
+  content.render(this.content);
+
+  this.modal.append(this.content);
+
+  this.modal.addEventListener('click', (event) => {
+    if (event.target === this.modal) {
+      this.modal.remove();
+    }
+  });
+
+  parent.append(this.modal);
+};
```

Comment:
add close() method, that would allow to hide modal window outside of the component.